src/Templater.js

```diff
@@ -94,7 +94,7 @@ export default class Templater {
         const find = key =>
             language_group =>
                 _.includes(language_group.countries, key)
-                || language_group.countries[key];
+                || language_group.countries[_.toUpper(key)];
```

Comment:
@pandafox How about we restrict the toUpper() only here so we don't affect other use cases?

Reply:
I think that'll work, but it'll mean we need to convert multiple times for the same value.
Prob more efficient if we just convert to 2 chars.

As for testing, we can just test the value of _language after calling derived_language with
1. lower case 2 letter
2. upper case 2 letter
3. all lower case country name
4. proper case country name

etc.,

Reply:
hmmm I think we can do that using spies. I'll see what I can do.